src/config/settings.py

```diff
@@ -36,6 +36,13 @@
     }
 }
 
+# Database role that public repos grant access to
+# All datahub users are granted access to this role
+PUBLIC_ROLE = 'dh_public'
+PUBLIC_ROLE_EMAIL = 'noreply_public@datahub.csail.mit.edu'
```

Comment:
Change noreply_public and noreply_anon to noreply+public and noreply+anon so they both go to noreply@datahub.csail.mit.edu. That's the address we have configured that's guaranteed to never see mail sent to it.